charts/prometheus/values.yaml

```diff
@@ -15,7 +15,7 @@ serviceAccounts:
     name:
     annotations: {}
 
-alertmanager:
+old_alertmanager:
```

Comment:
Variablenames sould be camel case.

Why is this needed anyway?

The condition in chart.yaml is "alertmanager.enabled".

Reply:
@monotek 
There must be some glitch on Github since I can't find `old_alertmanager` in values.yaml.